prompt_toolkit/shortcuts/prompt.py

```diff
@@ -846,11 +873,17 @@ def run_sync() -> Any:
                 self.default_buffer.reset(Document(default))
                 return await self.app.run_async(pre_run=pre_run2)
 
-        if async_:
+        if _async:
             return ensure_future(run_async())
         else:
             return run_sync()
 
+    async def prompt_async(self, *a: Any, **kw: Any) -> Any:
+        """
+        Like the `prompt()` method, but return a coroutine.
+        """
+        return await self.prompt(_async=True)
```

Comment:
The `*a` and `**kw` args are not used.

Reply:
Thanks for pointing out @bjorn3, these would need to be passed along to ``self.prompt``.

I'm still not sure though, whether I want to use ``*args`` and ``**kwargs``, instead of repeating all the arguments. Mypy can't infer types in case of ``*args`` and ``**kwargs`` like here.